supply-chain-security/compromises/2013/apt.md

```diff
@@ -0,0 +1,13 @@
+# APT didn't enforce signature validation for source packages
+
+It was discovered that APT's crypto-based routines to verify binary package authenticity wasn't being used for source packages.
+
+## Impact
+
+* A CVE was assigned
+* APT was fixed via a patch
+* Discussion followed around further supply chain concerns
+
+## Type of compromise
+
+Insufficient client-side package authenticity verification
```

Comment:
I think the type of compromise for this would be `(Negligence)`, @SantiagoTorres do you agree?  If so, we need to update the `(Negligence)` definition to include "lack of adherence to best practices"

Reply:
@SantiagoTorres